engine/OD/od_angular.h

```diff
@@ -28,7 +28,7 @@ typedef struct{
 // deg
 static inline float AngularOD_PositionTo_deg(angular_position_t self)
 {
-    return self.private;
+    return self;
```

Comment:
In this case you need to return a float so you have to return `self.private`.
`self.private` => float
`self` => angular_position_t
So you have to use the good value depending on the value type you want to use.

Reply:
got it, Will do the necessary.